packages/docs/src/content/components/previous-next/index.mdx

```diff
@@ -1,17 +1,33 @@
 ---
 title: Previous and Next
 navigationLabel: Previous & Next
-description: A way for users to navigation into previous or next pages of content.
+description: A way for users to navigate to previous or next pages in a sequence of content.
+inpagenav: true
 template: detail
+storybook: ?path=/story/components-previous-next
```

Comment:
The `storybook` prop for action-banner is just the component name, and not the full path - should they be consistent?

Reply:
yep - sorting that on action-banner branch now